README.md

```diff
@@ -38,26 +36,22 @@ machine1 = do
   guard (number /= 0)
   let scaled = div number 2
   pure $ show scaled
+  
+result :: Array String
+result = toUnfoldable unit machine1
 ```
 
 This will create a machine `machine1` which goes through the "inputs"
 from the array. It then checks and halts on any zero input, and otherwise
 scales the inputs (by dividing by 2). The result is then transformed into a string.
 
-The resulting machine can be materialized via
-
-```purescript
-> toUnfoldable unit machine1 :: Array String
-["5","10","15","20","25"]
-```
+The resulting machine can be materialized via `toUnfoldable unit machine1 :: Array String`.
```

Comment:
Maybe this could be moved to a comment above `result` in the code snippet instead of here.